XMLtoTree.cpp

```diff
@@ -0,0 +1,184 @@
+#include <iostream>
+#include <string.h>
+#include <vector>
+#include <stack>
+#include <cctype>
+#include <algorithm>
+
+using namespace std;
+
+class treeNode{
+    public :
+    string identifier;
+    string content;
+    treeNode * parent; // for navigating back from child to parent
+    vector <treeNode *> children;
+    long long level;
+
+    treeNode()
```

Comment:
do you plan to insert initializations in the constructor?

Reply:
Left for future improvements